assets/html/about.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Code for San José - About</title>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="../css/global.min.css" />
+  <link rel="stylesheet" href="../css/header.min.css" />
+  <link rel="stylesheet" href="../css/footer.min.css" />
+  <link rel="stylesheet" href="../css/about.min.css" />
+  <script src="https://kit.fontawesome.com/a5c8bf6536.js" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i&display=swap"
+    rel="stylesheet" />
+  <script src="../js/app.js"></script>
+</head>
+
+<body>
+  <div class="container">
+    <header>
+      <a href="../../index.html">
+        <img src="../img/logonav.png" id="navlogo" alt="Code for San José logo" />
+      </a>
+      <div id="menu-btn">
+        <i onclick="menu_toggle()" id="fa-btn" class="fas fa-2x fa-bars"></i>
+      </div>
+      <div id="navlinks">
+        <div id="links">
+          <a id="homeLnk" href="../../index.html">Home</a>
+          <a id="volLnk" href="./volunteer.html">Volunteer</a>
+          <a id="abtLnk" href="./about.html">About Us</a>
+          <a id="projLnk" href="./projects.html">Projects</a>
+          <a id="evLnk" href="https://www.meetup.com/Code-for-San-Jose/" target="_blank" rel="noopener noref">Events</a>
+          <a id="donLnk"
+            href="https://secure.codeforamerica.org/page/contribute/donate-to-a-brigade-today?source_codes=Brigade-page&brigade=Code%20for%20San%20Jose"
+            target="_blank" rel="noopener noref">Donate</a>
+          <a id="faqLnk" href="./faq.html">FAQ</a>
+        </div>
+        <div id="sociallinks">
```

Comment:
`id`s are kind of difficult to read without a dash. Instead of 'donLnk'  can it be 'donation-link'?

Reply:
These ID's ended up being unnecessary and I've removed them from all headers. That being said, I do tend to use camel case a lot, and my consistency on that could use some improvement. Would you like all ids and classes to have consistent use of dash separators on the site? If so, I think we should make that an issue and fix after launch as it will take some time to coordinate between the HTML and SCSS files.

Reply:
Yes, it doesn't need to be fixed asap